toast.yml

```diff
@@ -94,7 +94,7 @@ tasks:
       . $HOME/.cargo/env
       cargo clippy --all-targets --all-features -- \
         --deny warnings --deny clippy::all --deny clippy::pedantic
-      cargo fmt --all -- --check
+      cargo fmt --all -- --check || echo "ERROR: Please correct the formatting errors above"
```

Comment:
Is this because it was difficult to understand the output? I wonder if we should do this for all the lint checks (or none of them).

Either way, a few minor suggestions:
- `echo` to standard error rather than standard out (e.g., `echo '...' 1>&2`)
- Punctuate the sentence.
- Use single quotes unless we need the interpolation capability of double quotes. I know this is nitpicky, but I find that this practice avoids mistakes in which Bash interprets some part of the string that you didn't intend it to (though here I think it's inconsequential).

Reply:
Yes, when `cargo fmt` fails, it just prints a`diff` and I didn't know why or even what command was triggering that.
Whereas `clippy` has some explanation like `error: unneeded return statement`, so I knew how to address that.